### PR TITLE
Dg update command

### PR DESCRIPTION
**Pull request by dtaywd**
Added UML diagrams for UpdateCommand. NewCommand UML diagram was also edited to reflect the new expiry date added to item object.

### CODE REVIEW

**Comment by awesomesjh** [comment]
The sequence diagram of the NewCommand needs to be updated as well.

File: docs/uml-diagrams/UpdateCommand.puml [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+deactivate Inventory
+
+alt newQuantity == -1
+    UpdateCommand -> Item: getQuantity()
+    activate Item #fbffb2
+    Item --> UpdateCommand: oldItemQuantity
+    deactivate Item
+    UpdateCommand -> UpdateCommand: newQuantity = oldItemQuantity
+end
+alt newPrice == -1
```
Comment (awesomesjh) [inline docs/uml-diagrams/UpdateCommand.puml:74]:
```suggestion
opt newPrice == -1
```
Same issue as above. Remember to update the png as well.

**Comment by dtaywd** [comment]
> The sequence diagram of the NewCommand needs to be updated as well.

Ok will do

File: docs/uml-diagrams/UpdateCommand.puml (hunk 1 above)
Comment (dtaywd) [reply docs/uml-diagrams/UpdateCommand.puml]: Ok will do. Thanks for catching my mistake!

File: docs/uml-diagrams/UpdateCommand.puml [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    UpdateCommand -> Item: getPrice()
+    activate Item #fbffb2
+    Item --> UpdateCommand: oldItemPrice
+    deactivate Item
+    UpdateCommand -> UpdateCommand: newPrice = oldItemPrice
```
[the remark this answers is not in the document]
Comment (dtaywd) [reply docs/uml-diagrams/UpdateCommand.puml]: Have removed the following lines.

**Comment by rismm** [comment]
Good use of UML diagrams, looks good to me. Thanks for the work!

**Pull request merged**